pkg/slack/slack.go

```diff
@@ -310,6 +310,18 @@ func BuildJobParams(params string) (map[string]string, error) {
 	}
 	jobParams := make(map[string]string)
 	for _, combinedParam := range splitParams {
+		// Check for DEVSCRIPTS_CONFIG ; if found, we know we're dealing with semicolon separated parameters.
```

Comment:
I don't think it's really necessary to lock this logic down to only `DEVSCRIPTS_CONFIG`.  I'd much rather it be generic so that if/when another case comes along, we don't have to add another condition to the code.  What's important is the syntax of how to pass these variables into the ClusterBot.